README.md

```diff
@@ -62,16 +62,18 @@ ansible-playbook test/demo/organization.yml
 Set up debugging using ansible's test-module
 
 ```sh
-ansible-playbook debug-setup.yml
+make debug-setup
 ```
 
 Debug with ansible's test-module
 
 ```sh
-./ansible/hacking/test-module -m <path to ansible module> -a @<path to input arguments to module> -D <path to debugger>
+MODULE=<module name> DATA=<data file name> make debug
 
-# Example: debug the activation_key module with pdb
-./ansible/hacking/test-module -m ./foreman-ansible-modules/modules/katello_activation_key.py -a @activation-key-args.json -D /usr/lib/python2.7/site-packages/pdb/pdb.py
+# Example: debug the katello_content_view module
+$ MODULE=katello_content_view DATA=content-view make debug
```

Comment:
Do these have to be before the `make` command? Given `make` is the entry point, I think it atypical for most to start with the environment variables. I'm more prone to start typing `make debug` and then include my options. If these could possibly be positional (at least module) that would be great too.

Reply:
Oh, they can be after the command, like this:

```sh
make debug MODULE=katello_content_view DATA=content-view
```

I wonder if we should just go ahead and lowercase the variables. Make doesn't really support positional arguments because the syntax is `make <target> <target> ...`. There are hacky ways of making it work, but I'd avoid it.

Reply:
Not if you meant that by 'hacky', but you can use wildcard in the targets. It would allow for something like
`make debug-katello-product`.

Reply:
@mdellweg an interesting solution. Would it be bad to implement both?

Reply:
I guess it would not hurt.